content_handler/runtime_holder.h

```diff
@@ -65,7 +65,7 @@ class RuntimeHolder : public blink::RuntimeDelegate,
   std::string DefaultRouteName() override;
   void ScheduleFrame(bool regenerate_layer_tree = true) override;
   void Render(std::unique_ptr<flow::LayerTree> layer_tree) override;
-  void UpdateSemantics(std::vector<blink::SemanticsNode> update) override;
+  void UpdateSemantics(blink::SemanticsNodeUpdates update) override;
```

Comment:
Why copy the collection in this call and not pass it by reference instead?

Reply:
I realize you did not add this call. So it fine for us to address this in a future patch.